<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="rbSha1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SHA-1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbSha256">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SHA-256</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbSha512">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SHA-512</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>421</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter files by extensions:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputExtensions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Extensions list separated by spaces</string>
    </property>
    <property name="placeholderText">
     <string>log jpg docx mkv</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ignoreDbFiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Exclude *.ver.json files from the filelist</string>
    </property>
    <property name="text">
     <string>Ignore Veretino DB files</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ignoreShaFiles">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Exclude checksum summary files from the filelist</string>
    </property>
    <property name="text">
     <string>Ignore sha files (*.sha1/256/512)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonIgnore">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>96</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ignore</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonIncludeOnly">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>10</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Include Only</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>421</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="inputJsonFileNamePrefix">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Input a prefix for the DB filename. Default: checksums</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
